src/lib/parser/parser.ts

```diff
@@ -32,70 +32,42 @@ export class AuctionParser {
 		return processedMsg;
 	}
 
-	private wordIsAuctionType(word: string) {
-		const wtsAuctionTypes = ['WTS', 'SELLING'];

```

Comment:
There's admittedly no replacement for "SELLING/BUYING" (my code only checks WTS/WTB) but I am also not sure I've EVER seen that spelled out, so ...

Reply:
it happens pretty regularly unfortunately, but I'm sure we can extend your parser to handle that too.